### DISCUSSION TITLE
Extend height limit

### DISCUSSION THREAD

**Discussion by The-Ultimate-Dart-Monkey** [Minestom/Minestom#926] [Help] [answered]
![image](https://user-images.githubusercontent.com/41764863/163635238-46eeb599-e023-40ef-884a-b4b4a2062183.png)
im trying to create a -512 to 512 world I'm confused on the different of these three

**Comment by TheMode** [comment, marked as answer]
See https://minecraft.fandom.com/wiki/Custom_dimension
`height`
> The number of blocks in the layer.

`logical_height`
> The maximum height to which [chorus fruits](https://minecraft.fandom.com/wiki/Chorus_fruit) and [nether portals](https://minecraft.fandom.com/wiki/Nether_portal) can bring players within this dimension. This excludes portals that were already built above the limit as they still connect normally. May not be greater than height.

`min_y`
>The minimum height in which blocks can exist within this dimension. Should be between -2032 and 2031 and be a multiple of 16 (effectively making 2016 the maximum). Setting it lower than -2048 will only allow the temporary placement of blocks below it as they won't be saved. Furthermore, lighting won't work correctly at Y-coordinate -2048 and below.[[1]](https://minecraft.fandom.com/wiki/Custom_dimension#cite_note-2)

In this case you want `height= 1024`, and `min_y= -512`

**Comment by The-Ultimate-Dart-Monkey** [comment]
alr thx explaining it
